I think it also needs `fakeRobotConfig.put("allow-deprecated-devices", 1);`, otherwise it quits with an error:
```
[1623682245.250370] [ERROR] /home/mfussi/iit/yarp/src/libYARP_dev/src/yarp/dev/PolyDriver.cpp:321 bool yarp::dev::PolyDriver::coreOpen(yarp::os::Searchable&) (0x000d7051) |yarp.dev.PolyDriver| Device "fakeMotor" is deprecated. Pass the "allow-deprecated-devices" option in the configuration if you want to open it anyway.
```

I just tested the program with the renamed device and the above config option, and everything seems to work fine.

--- a/src/TutorialModule.cpp
+++ b/src/TutorialModule.cpp
@@ -70,7 +70,7 @@ bool TutorialModule::configure(yarp::os::ResourceFinder &rf)
     // open a dummy fake robot
     yarp::os::Property fakeRobotConfig;
     fakeRobotConfig.put("device",    "controlboardwrapper2");
-    fakeRobotConfig.put("subdevice", "fakeMotor");
+    fakeRobotConfig.put("subdevice", "fakeMotionControl");
     fakeRobotConfig.put("name", "/fakeRobot/head");
     fakeRobot.open(fakeRobotConfig);
 

